packages/graphql-api-commons/src/authentication/types.ts

```diff
@@ -13,10 +13,12 @@ export interface IAuthenticator<Credentials, Identity> {
 	login(credentials: Credentials): Promise<Identity | null>
 
 	/**
-	 * Log iinto the system with a JWT token
+	 * Log into the system with an Authorization header
```

Comment:
Let'es leave this as `verifyToken` - token implies that you have a bit of text in some undefined format. `Authorization` implies that you have an authorization header. Also, `verifyAuthorization` could simply mean logging in.

Reply:
Just updated the PR with the token extract for the `IdentityRequestContext`